test/functional/wallet_avoidreuse.py

```diff
@@ -58,78 +58,85 @@ def assert_unspent(node, total_count=None, total_sum=None, reused_supported=None
         assert_approx(stats["reused"]["sum"], reused_sum, 0.001)
 
 def assert_balances(node, mine):
-    '''Make assertions about a node's getbalances output'''
+    """Make assertions about a node's getbalances output."""
     got = node.getbalances()["mine"]
     for k,v in mine.items():
         assert_approx(got[k], v, 0.001)
 
 class AvoidReuseTest(BitcoinTestFramework):
-
     def set_test_params(self):
         self.setup_clean_chain = False
         self.num_nodes = 2
+        # This test isn't testing txn relay/timing, so set whitelist on the
+        # peers for instant txn relay. This speeds up the test run time 2-3x.
+        self.extra_args = [["-whitelist=127.0.0.1"]] * 2
 
     def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
 
     def run_test(self):
-        '''Set up initial chain and run tests defined below'''
-
+        """Set up initial chain and run tests defined below."""
         self.test_persistence()
         self.test_immutable()
 
         self.nodes[0].generate(110)
         self.sync_all()
         reset_balance(self.nodes[1], self.nodes[0].getnewaddress())
-        self.test_fund_send_fund_senddirty()
+        self.test_fund_send_fund_send_dirty()
         reset_balance(self.nodes[1], self.nodes[0].getnewaddress())
         self.test_fund_send_fund_send()
 
     def test_persistence(self):
-        '''Test that wallet files persist the avoid_reuse flag.'''
-        # Configure node 1 to use avoid_reuse
-        self.nodes[1].setwalletflag('avoid_reuse')
+        """Test that wallet files persist the avoid_reuse flag."""
+        self.log.info("Test wallet files persist avoid_reuse flag")
```

Comment:
s/Test wallet files/Test that wallet files

Reply:
Thanks @jachiang. I removed both articles from the docstring ("that" / "the") for the log version to keep it shorter, as it was longer than the other logs, and for consistency because the other added logs don't have articles either.